modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1093,11 +1093,28 @@ protected function render_tab_titles_html( $item_settings ): string {
 			'style' => '--n-tabs-title-order: ' . $item_settings['tab_count'] . ';',
 		] );
 
-		$render_attributes = $this->get_render_attribute_string( $setting_key );
-		$text_class = $this->get_render_attribute_string( 'tab-title-text' );
-		$icon_class = $this->get_render_attribute_string( 'tab-icon' );
+		$rendered_icons = $this->render_tab_icons_html( $item_settings );
 
-		$icon_html = Icons_Manager::try_get_icon_html( $item_settings['item']['tab_icon'], [ 'aria-hidden' => 'true' ] );
+		ob_start();
+		?>
+			<button <?php $this->print_render_attribute_string( $setting_key ); ?>>
+				<?php echo $rendered_icons; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+				<span <?php $this->print_render_attribute_string( 'tab-title-text' ); ?>>
+					<?php echo wp_kses_post( $title ); ?>
+				</span>
+			</button>
+		<?php
+		return ob_get_clean();
+	}
+
+	protected function render_tab_icons_html( $item_settings ): string {
+		$icon_settings = $item_settings['item']['tab_icon'];
+
+		if ( empty( $icon_settings['value'] ) ) {
```

Comment:
This was the main fix for this PR. 
On the front end the icon wrapper was still added to the DOM, even when there were no icons.